scripts/appveyor-tool.ps1

```diff
@@ -129,6 +129,7 @@ Function InstallRtools {
     $gcc_path = $env:GCC_PATH
   }
   $env:PATH = $RtoolsDrive + '\Rtools\bin;' + $RtoolsDrive + '\Rtools\MinGW\bin;' + $RtoolsDrive + '\Rtools\' + $gcc_path + '\bin;' + $env:PATH
+  $env:BINPREF=$RtoolsDrive + '/Rtools/mingw_$(WIN)/bin/'
```

Comment:
[this](https://ci.appveyor.com/project/jimhester/r-appveyor/build/1.0.11/job/20m4orjg4arjtmtj#L423) is compiling both archs. During the actual build of the devel package you set `--no-multiarch`, so there it is only compiling one arch (and running one set of tests).

Sorry about the whitespace issues, thought I had set my global autocrlf settings appropriately on my windows VM, but apparently not.

https://github.com/rwinlib/r-base/wiki/Testing-Packages-with-Experimental-R-Devel-Build-for-Windows#rterm--rgui has documentation on running packages with the new toolchain. All that should be necessary to add to the path is `Rtools/bin`, not the specific mingw_?? directories.